aries_cloudagent/messaging/valid.py

```diff
@@ -194,7 +194,7 @@ class IndyDID(Regexp):
     """Validate value against indy DID."""
 
     EXAMPLE = "WgWxqztrNooG92RXvxSTWv"
-    PATTERN = rf"^(did:sov:)?[{B58}]{{21,22}}$"
+    PATTERN = rf"^(did:sov:|did:peer:)?[{B58}]{{21,22}}$"
```

Comment:
https://identity.foundation/peer-did-method-spec/index.html#matching-regex

Not sure whether indy-sdk would take these though

Reply:
but note that it's bad regex; the trailing $) characters appear twice. Better:
^did:peer:[01](z)([1-9a-km-zA-HJ-NP-Z]{46,47})$